Formula/clickhouse-odbc.rb

```diff
@@ -0,0 +1,48 @@
+class ClickhouseOdbc < Formula
+  desc "ClickHouse ODBC driver"
+  homepage "https://clickhouse.yandex"
+  url "https://github.com/yandex/clickhouse-odbc.git",
+    :tag => "v1.0.0.20180903",
+    :revision => "17ecca3993e164d2d49360ed9be68becc11b7b05"
+  # head "https://github.com/yandex/clickhouse-odbc.git"
+
+  bottle do
+    # sha256 "256" => :high_sierra
+    # sha256 "256" => :sierra
+    # sha256 "256" => :el_capitan
+  end
+
+  # option "with-debug", "Install debugging version with advanced logs"
+
+  depends_on "cmake" => :build
+  depends_on "libiodbc" => :build
+  depends_on "libtool" => :build
+  depends_on "ninja" => :build
+  depends_on "openssl" => :build
+
+  needs :cxx14
+
+  def install
+    args = std_cmake_args
+    # args << "-DCMAKE_BUILD_TYPE=Debug" if build.with? "debug"
```

Comment:
Remove

Reply:
maybe uncomment? debug can be useful sometimes

Reply:
Sorry, no options. If you can make it into debug by default, and people don't mind the extra logging, why not. (Ideally, it should be configurable at run time!)